hsm-client/hsm-client_install.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh -x
+
+## Script to install hsm-client service
+
+# Install Amazon SSM agent
+sudo yum install -y https://s3.amazonaws.com/ec2-downloads-windows/SSMAgent/latest/linux_amd64/amazon-ssm-agent.rpm
+
+# pip is not available in CentOS 7 core repositories there is a requirement to enable EPEL repositories prior
+sudo yum --enablerepo=extras install -y epel-release
+sudo yum install -y python-pip
+
+# Adding in netcat and jq for troubleshooting
+sudo yum install -y nmap-ncat jq
+
+# Download & install AWS-CLI
+sudo pip install awscli
+
+# Install HSM client, the SDK and dependencies
+sudo yum install -y https://s3.amazonaws.com/cloudhsmv2-software/CloudHsmClient/EL7/cloudhsm-client-latest.el7.x86_64.rpm
+sudo yum install -y https://s3.amazonaws.com/cloudhsmv2-software/CloudHsmClient/EL7/cloudhsm-client-pkcs11-latest.el7.x86_64.rpm
```

Comment:
My reading of the docs suggests you don't need the pkcs11 or dyn RPMs.  Have I missed something?  We used to install the pkcs11 library back when we were trying to get things going with Python, but I think everything in Java land is handled by the jce RPM below.

Reply:
Done